.github/workflows/source_aws.yml

```diff
@@ -145,14 +145,17 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
-      - name: Checkout
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
       - name: Set up Go 1.x
         uses: actions/setup-go@v3
         with:
           go-version-file: plugins/source/aws/go.mod
           cache: true
           cache-dependency-path: plugins/source/aws/go.sum
+      - name: Get dependencies
```

Comment:
Won't `go build` download dependencies? Maybe we should remove it from all workflows

Reply:
I made changes to the `policies` workflow so it matched 100% the other workflow... when investigating why the build was taking so long... I can revert all of the changes to that workflow

Reply:
How about we do it in another PR? One that aligns all our workflows?